### DISCUSSION TITLE
Настройка Вида

### DISCUSSION THREAD

**Discussion by Dmitry0077** [quarckster/kodi.kino.pub#174] [Ideas]
Друзья, вы супер, но есть вопрос к виду представления контента - ни в одном из вариантов вида (Список, Постер и бла бла) не видно ГОД фильма нормально, СОВСЕМ, даже в описании отдельной строкой. Есть длинные названия фильмов, где год в самом названии теряется. Это совсем не гуд..

В идеале, при любом виде СРАЗУ должны быть видны:
1. Год
2. Синопсис
3. Рейтинги
4. Доступное качество

Благодарю заранее за услышанное пожелание.

**Comment by quarckster** [comment]
Здравствуйте, спасибо за пожелание, не могли бы вы присласть картинку, как вы это видите в разных вариантах отображения. Я посмотрю, насколько это выполнимо.

**Reply by Dmitry0077** [reply]
Добрый день.

Примерно так, наверное (аттач). "Стену" не визуализировал, она сейчас и так с нулевой информативностью. В Shift - два варианта.
![Постер](https://user-images.githubusercontent.com/79451318/108863570-015bcf80-75fa-11eb-9b64-7deb8bdfde32.jpg)
![Список](https://user-images.githubusercontent.com/79451318/108863578-03259300-75fa-11eb-8a35-62d917f5ec39.jpg)
![Стена информации](https://user-images.githubusercontent.com/79451318/108863580-03259300-75fa-11eb-95c7-0426cda4c2cf.jpg)
![Фанарт](https://user-images.githubusercontent.com/79451318/108863582-03be2980-75fa-11eb-8b39-70f2625d84ed.jpg)
![Широкий список](https://user-images.githubusercontent.com/79451318/108863584-0456c000-75fa-11eb-8b70-aa997a9f0f2a.jpg)
![Shift](https://user-images.githubusercontent.com/79451318/108863587-04ef5680-75fa-11eb-9b25-5e0e4c0d32fe.jpg)


Благодарю

**Reply by quarckster** [reply]
Да, думаю, это вполне выполнимо и не составит большого труда.

**Comment by Dmitry0077** [comment]
И еще вопрос: на сколько я понимаю, метаданные из скриншота никак не реально подтягивать? Это здорово бы упростило)
![Screenshot_20210223-172349](https://user-images.githubusercontent.com/79451318/108866051-82b46180-75fc-11eb-9bb2-8a10474679ed.png)

**Reply by quarckster** [reply]
Что упростило бы? Метаданные в плагине берутся из API кинопаба. В вашем скриншоте показана локальная библиотека.

**Comment by Dmitry0077** [comment]
В смысле не пришлось бы перегружать интерфейс деталями, если б метаданные
подтягивались полностью (длительность, формат видео, звука), как в
библиотеке. Как минимум длительность подтягивается, вот я и подумал, что
остальное тоже может подтянуться, возможно..

чт, 25 февр. 2021 г., 15:58 Dmitrii Misharov <notifications@github.com>:

**Comment by Dmitry0077** [comment]
Супер!!! Спасибо огромное!

чт, 25 февр. 2021 г., 16:00 Dmitrii Misharov <notifications@github.com>:
